integration-tests/opentelemetry-jaeger-remote/src/test/java/io/quarkus/it/opentracing/OTelJaegerRemoteTest.java

```diff
@@ -0,0 +1,119 @@
+package io.quarkus.it.opentracing;
+
+import static com.github.dockerjava.api.model.HostConfig.newHostConfig;
+
+import java.time.Duration;
+
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.jr.ob.JSON;
+import com.fasterxml.jackson.jr.stree.JacksonJrsTreeCodec;
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.CreateContainerResponse;
+import com.github.dockerjava.api.model.ExposedPort;
+import com.github.dockerjava.api.model.Ports;
+import com.github.dockerjava.core.DockerClientBuilder;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import jakarta.inject.Inject;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.awaitility.Awaitility;
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class OTelJaegerRemoteTest {
+
+    private static final OkHttpClient client = new OkHttpClient();
+    private static final int QUERY_PORT = 16686;
+    private static final int COLLECTOR_PORT = 14250;
+    private static final int HEALTH_PORT = 14269;
+    private static final String JAEGER_URL = "http://localhost";
+    @Inject
+    OpenTelemetry openTelemetry;
+    private static final DockerClient dockerClient = DockerClientBuilder.getInstance().build();
```

Comment:
please build the project for the automatic formater to kick in... Static attributes must be declared before injections.

Reply:
@brunobat I built the project using the command mvn process-sources as seen in the doc for formatting but I didn't get any errors. Can you please share the mvn command or ref in the doc for formatting?

Reply:
The `mvn clean install` will do it automatically. The recommendation I did still applies: static before inject

Reply:
Done